config/text.json

```diff
@@ -11,7 +11,7 @@
         "ADD": "joined your team!"
     },
     "TEAM": {
-        "ENTRY": "TEAM MENU",
+        "ENTRY": "MANAGE TEAM",
         "EXIT": " "
```

Comment:
No need for the "EXIT" text here, it's just an empty string anyways. If you wanted an empty line there are probably better, more reusable ways of doing that

Reply:
Deleted `"EXIT": " "`